Apple can't find Fortran compiler
```
CMake Error at CMakeLists.txt:11 (project):
  No CMAKE_Fortran_COMPILER could be found.
  Tell CMake where to find the compiler by setting either the environment
  variable "FC" or the CMake cache entry CMAKE_Fortran_COMPILER to the full
  path to the compiler, or to the compiler name if it is in the PATH.
```

Linux has logs which are too long
```
The log length has exceeded the limit of 4 MB (this usually means that the test suite is raising the same exception over and over).

The job has been terminated
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -138,6 +138,9 @@ install:
   - if [[ "`uname`" == "Darwin" ]]; then pip3 install requests; fi
   - if [[ "`uname`" == "Linux" ]]; then travis_retry pip3 install --user requests; fi
 
+  # Install GCC and gfortran
+  - if [[ "`uname`" == "Darwin" ]]; then travis_retry brew install gcc; fi
+
   # Install latest Polly toolchains and scripts
   - wget https://github.com/ruslo/polly/archive/master.zip
   - unzip master.zip

--- a/jenkins.py
+++ b/jenkins.py
@@ -132,6 +132,12 @@ def run():
   ):
     verbose = False
 
+  if (
+      os.getenv('TRAVIS') and
+      (project_dir == 'examples/SuiteSparse-dynLAPACK')
+  ):
+    verbose = False
+
   project_dir = os.path.join(cdir, project_dir)
   project_dir = os.path.normpath(project_dir)
 

